### PR TITLE
Fix for KW critical issues.

### PR DESCRIPTION
**Pull request by rbabuche**
Tracked On: OAM-76865
Signed-off-by: Raveendra Babu Chennakesavulu
			<raveendra.babu.chennakesavulu@intel.com>

### CODE REVIEW

**Comment by rbabuche** [comment]
@JeevakaPrabu

File: lib/driver_if.cpp [hunk 1]
```diff
@@ -1889,33 +1865,18 @@ int driver_set_country_code(void *handle, const char *code)
...
-		return -1;
-	}
-}
-
 static void event_handler_sock(struct nl_sock *sock)
 {
-	struct nl_cb *cb = nl_socket_get_cb(sock);
+	struct nl_cb *cb = NULL;
+	
+	if(sock !=NULL){
```
Comment (JeevakaPrabu) [inline lib/driver_if.cpp]: -Introduce space between if and (
-Introduce space between != and NULL
-Introduce space between ( and {

**Review by JeevakaPrabu — COMMENTED** [review]
Android.mk file changes missing

**Comment by rbabuche** [comment]
@JeevakaPrabu  review comments are addressed, pls re-review the changes.

**Review by JeevakaPrabu — APPROVED** [review]

**Comment by rbabuche** [comment]
@sgnanase @swaroopbalan pls merge the changes

**Pull request merged**